react/AppLinker/index.jsx

```diff
@@ -107,19 +115,36 @@ export class AppLinker extends React.Component {
       }
     } else if (isMobile() && appInfo) {
       // If we are on the "mobile web version", we try to open the native app
-      // if it exists. If it fails, we redirect to the web version of the
-      // requested app
-      onClick = this.openNativeFromWeb
+      // if it exists with an universal links. If it fails, we redirect to the web
+      // version of the requested app
+      // Only on iOS ATM
+      if (isAndroid()) {
+        onClick = this.openNativeFromWeb
+      } else {
+        href = generateUniversalLink({ slug, nativePath, fallbackUrl: href })
+      }
     }
 
     return children({ ...appInfo, onClick: onClick, href })
   }
 }
 
+AppLinker.defaultProps = {
+  nativePath: '/'
+}
 AppLinker.propTypes = {
+  //Slug of the app : drive / banks ...
   slug: PropTypes.string.isRequired,
-  href: PropTypes.string.isRequired
+  /*
+  Full web url : Used by default on desktop browser
+  Used as a fallback_uri on mobile web 
+  */
+  href: PropTypes.string.isRequired,
+  /*
+    Path used for "native link"
+  */
+  nativePath: PropTypes.string
 }
 
 export default AppLinker
-export { NATIVE_APP_INFOS }
+export { NATIVE_APP_INFOS, getUniversalLinkDomain, generateUniversalLink }
```

Comment:
Why are we exporting the universal link helpers from this component ?

Reply:
It seems easier to do: 
import AppLinker, {getUniversalLinkDomain} from 'cozy-ui/AppLinker'  

than : 
import AppLinker from 'cozy-ui/AppLinker'  
import {getUniversalLinkDomain} from 'cozy-ui/Applinker/native'

Reply:
But is `getUniversalLinkDomain ` going to be used outside of cozy-ui? If yes we can leave it as it is :)

Reply:
Yes, it will. (in the mobile handleDeeplink process)